### ISSUE TITLE
Frankenmuth Issue unloading at East Grain Pool and CS not waiting for other truck to unload first

### ISSUE THREAD

**Issue by James31283** [Courseplay/Courseplay_FS22#2359]
**READ FIRST**
https://github.com/Courseplay/Courseplay_FS22/wiki

**Steps to reproduce**
Send any truck to the East Grain Pool to unload goods for storage or rail transit on Frankenmuth Map

**Actual behavior**
Truck stops beyond road intersection and then stops course play, when manual drive to intersection, CS turns too late to enter unloading point and gets stuck. Second issue, if truck 1 is still unloading, truck 2 pushes truck 1 to get it out of the way.

**Expected behavior**
Stop and make normal turn at intersection and enter the unloading point for the location with plenty of room with trailer. Second issue truck 2 waits for truck 1 to finish unloading and gets out of the way before proceeding. 

**Screenshots**
Youtube Video link: https://youtu.be/VKqfrbPTjuU

**Courseplay version**
Version used: 7.2.0.2

**Log**
Please attach a log with all debug channels active.
For help with the debug channels,
please check out the readme at the github front page.

**Map**
Frankenmuth Map

**Vehicles/ implements involved**
Which vehicle or implement?  volvo - fh16 & krampe - sks30_150
Mod yes/no ? No

**Relevant script mods**
Mods like Autodrive or VCA and so on. No

**Comment by rohne83** [comment]
You're serious? You're using the Giants helper, not CP. Maybe the map has an issue, then ask the modder. This doesn't have to do anything with CP.

Issue can be closed.

**Comment by Tensuko** [comment]
What @rohne83 said.
Even if we have the option to drive, we only give the Task to Giants Helper, nothing else.

**Comment by James31283** [comment]
Thank you for the quick responses. While I appreciate the support and development of the CP team, the initial message was a little strong in tone to a novice user. 

I have only played FS with CP and almost refused to play FS without CP. I overlooked that the delivery mode did not have a CP, and I was wishful it could have been an easy CP fix. 

To my novice point above, I am only playing with CP installed. Would this be a good example of where others are using AutoDrive? If so, I would be happy to start exploring options other than the FS built-in helper.
